crashlog/crashutils.c

```diff
@@ -808,13 +808,17 @@ int process_info_and_error(char *filename, char *name) {
         snprintf(path, sizeof(path),"%s/%s", filename,tmp_data_name);
         snprintf(destion,sizeof(destion),"%s/%s", dir, tmp_data_name);
         do_copy_tail(path, destion, 0);
-        remove(path);
```

Comment:
Why need to check the return value of remove()? What's the consequence?

Reply:
CID 206964: (#2 of 2): Unchecked return value from library (CHECKED_RETURN)
5. check_return Calling remove(path) without checking return value. This library function may fail and return an error code. [Note: The source code implementation of the function has been overridden by a builtin model.]